### PR TITLE
Some fixes in pm4.

### PR DESCRIPTION
**Pull request by NgLamVN**
* Add missing permissions.
* Fix item give issues in command.

### CODE REVIEW

File: src/czechpmdevs/buildertools/commands/BlockInfoCommand.php [hunk 1]
```diff
@@ -48,11 +48,12 @@ public function execute(CommandSender $sender, string $commandLabel, array $args
         if(BuilderTools::getConfiguration()["items"]["blockinfo-stick"]["enabled"]) {
             $item = VanillaItems::STICK();
             $item->setCustomName(BuilderTools::getConfiguration()["items"]["blockinfo-stick"]["name"]);
-            $item->setNamedTagEntry(new ListTag(Item::TAG_ENCH, [], NBT::TAG_Compound));
-            $item->setNamedTagEntry(new ByteTag("buildertools", 1));
+            $nbt = $item->getNamedTag();
+            $nbt->setByte("buildertools", 1);
+            $nbt->setTag(Item::TAG_ENCH, new ListTag([], NBT::TAG_Compound));
+            $item->setNamedTag($nbt);
+            /*$item->addEnchantment(new EnchantmentInstance(EnchantmentIdMap::getInstance()->fromId(50), 1));*/
```
Comment (VixikHD) [inline src/czechpmdevs/buildertools/commands/BlockInfoCommand.php:55]: I think we do not need this commented out, github can show us how the code looked like in the past.

File: src/czechpmdevs/buildertools/commands/WandCommand.php [hunk 2]
```diff
@@ -44,8 +46,11 @@ public function execute(CommandSender $sender, string $commandLabel, array $args
         if(BuilderTools::getConfiguration()["items"]["wand-axe"]["enabled"]) {
             $item = VanillaItems::WOODEN_AXE();
             $item->setCustomName(BuilderTools::getConfiguration()["items"]["wand-axe"]["name"]);
-            /** @phpstan-ignore-next-line */
-            $item->addEnchantment(new EnchantmentInstance(EnchantmentIdMap::getInstance()->fromId(50), 1));
+            $nbt = $item->getNamedTag();
+			$nbt->setByte("buildertools", 1);
+			$nbt->setTag(Item::TAG_ENCH, new ListTag([], NBT::TAG_Compound));
+			$item->setNamedTag($nbt);
```
Comment (VixikHD) [inline src/czechpmdevs/buildertools/commands/WandCommand.php:52]: Wrong formatting

File: src/czechpmdevs/buildertools/commands/WandCommand.php [hunk 3]
```diff
@@ -44,8 +46,11 @@ public function execute(CommandSender $sender, string $commandLabel, array $args
...
+            /*$item->addEnchantment(new EnchantmentInstance(EnchantmentIdMap::getInstance()->fromId(50), 1));*/
```
Comment (VixikHD) [inline src/czechpmdevs/buildertools/commands/WandCommand.php:53]: This line is useless

**Review by VixikHD — CHANGES_REQUESTED** [review]

**Comment by NgLamVN** [comment]
Close due to I dont have enough time to fix that.

**Comment by NgLamVN** [comment]
Amost done.

**Comment by NgLamVN** [comment]
Github:
image
PHPStorm:
image

HOW ????

File: src/czechpmdevs/buildertools/commands/BlockInfoCommand.php [hunk 4]
```diff
@@ -50,9 +46,8 @@ public function execute(CommandSender $sender, string $commandLabel, array $args
             $item->setCustomName(BuilderTools::getConfiguration()["items"]["blockinfo-stick"]["name"]);
             $nbt = $item->getNamedTag();
             $nbt->setByte("buildertools", 1);
-            $nbt->setTag(Item::TAG_ENCH, new ListTag([], NBT::TAG_Compound));
             $item->setNamedTag($nbt);
-            /*$item->addEnchantment(new EnchantmentInstance(EnchantmentIdMap::getInstance()->fromId(50), 1));*/
```
Comment (VixikHD) [inline src/czechpmdevs/buildertools/commands/BlockInfoCommand.php]: I meant to remove the line. That custom enchantment should not be a thing anymore. I think

Comment (NgLamVN) [reply src/czechpmdevs/buildertools/commands/BlockInfoCommand.php]: I was try to implement it and it still work, its easier to see the distinguish between wand or block pick  stick and normal axe or stick

Comment (VixikHD) [reply src/czechpmdevs/buildertools/commands/BlockInfoCommand.php]: I think custom enchantments should be used for item with special power. I just wanted to add blessing effect to the item. Also I would like to keep the similarity with 3.0.0 branch.

**Pull request closed without merging**